src/Participation/LotteryFilter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace PhpAb\Participation;
+
+class LotteryFilter implements FilterInterface
+{
+    /**
+     * @float
+     */
+    private $propability;
+
+    /**
+     * @param int $propability The probability for the lottery in percent
+     *                         Should be 0 <=> 100
+     *                         0 is lowest probability for participation
+     *                         100 is the highest probability for participation
+     *
+     */
+    public function __construct($propability)
+    {
+        $this->propability = $propability;
+    }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function shouldParticipate()
```

Comment:
seems too complex. maybe I'm too stupid this afternoon. @waltertamboer hint for me?